src/_guides/language/concurrency/index.md

```diff
@@ -312,107 +311,144 @@ to implement isolates.
 
  [Flutter `compute()` function]: {{site.flutter-docs}}/cookbook/networking/background-parsing#4-move-this-work-to-a-separate-isolate
 
-
 ### Implementing a simple worker isolate
 
-This section shows the implementation for a
-main isolate and the simple worker isolate that it spawns.
-The worker isolate executes a function and then exits,
-sending the main isolate a single message as it exits.
-(The [Flutter `compute()` function][] works in a similar way.)
+These examples show the implementation of a
+main isolate that spawns a simple worker isolate.
+The examples use [`Isolate.run()`][], 
+which efficiently abstracts the complexity
+of setting up and managing worker isolates manually:
 
-This example uses the following isolate-related API:
+* Spawns (starts and creates) an isolate
+* Takes a function argument and executes
```

Comment:
This was hard for me to parse. These are steps that Isolate.run does for you right? It also sounds like we're saying it takes a function argument _after_ the isolate is spawned. 

Instead I'd probably say something like:
```suggestion
* Runs a function on the spawned isolate
```